sdk/python/toolchain/toolchain_test.go

```diff
@@ -347,6 +422,8 @@ func writePoetryToml(t *testing.T, path string) {
 	require.NoError(t, err)
 	fmt.Fprint(f, `[virtualenvs]
 in-project = true
+[virtualenvs.options]
+no-setuptools = true
```

Comment:
In Python 3.12 and later, this is always the case https://python-poetry.org/docs/configuration/#virtualenvsoptionsno-setuptools